plugins/plugin-chart-echarts/src/Timeseries/buildQuery.ts

```diff
@@ -37,6 +38,7 @@ export default function buildQuery(formData: QueryFormData) {
         timeCompareOperator(formData, baseQueryObject),
         sortOperator(formData, { ...baseQueryObject, is_timeseries: true }),
         rollingWindowOperator(formData, baseQueryObject),
+        resampleOperator(formData, { ...baseQueryObject, is_timeseries: true }),
```

Comment:
Not sure if it makes a difference, but should we place the resample operator before rolling window and time compare operators?

Reply:
nice catch!